app/assets/javascripts/comments.js

```diff
@@ -30,14 +30,14 @@ function init_comments_page() {
     });
     // end of removing comments
 
-
-    // comments autoload
-    var comment_page = 1;
-    load_comments();
-    // end of comments autoload
+    init_infinity_scroll(1, comments_source_url());
 
     // Helpers
 
+    function comments_source_url() {
+        return window.location.href + '/comments'
```

Comment:
We use named route helpers because we want to avoid of changing hardcoded urls across all application. So using such helpers should be a rule of thumb. I understand that this might be difficult and might look not needed in this particular case, but please try at  least.